languages/python/pyproject.toml

```
@@ -19,6 +19,18 @@ readme = "README.md"
 requires-python = ">=3.0"
 version = "1.0.0"
 
+[project.optional-dependencies]
+dev = [
+  "maturin >= 1.0,<2.0",
+  "uv >= 0.6,<1.0",
+]
+
+dev-linux = [
+  "maturin >= 1.0,<2.0",
+  "patchelf >= 0.17.0", # only needed for Linux
+  "uv >= 0.6,<1.0",
+]
```

```suggestion
[project.optional-dependencies]
dev = [
  "maturin >= 1.0,<2.0",
  "uv >= 0.6,<1.0",
]

dev-linux = [
  "patchelf >= 0.17.0", # only needed for Linux
]
```

Do you know if this would work and then you can call 
```
uv pip install .[dev]

if linux {
  uv pip install .[dev-linux]
}
```

Like does call pip install again remove the first install? Or are they additive? I feel like laying it out that way would be a little bit better than listing dependencies twice.